test_sort_numbers.py

```diff
@@ -0,0 +1,28 @@
+import sys
+from typing import List
```

Comment:
I am pretty sure that `import sys` and `from typing import List` are not needed, but they do not really cause any problems either. So I am fine with them staying here.